portal/src/graphql/adminapi/UserDetailsConnectedIdentities.tsx

```diff
@@ -230,22 +295,77 @@ const UserDetailsConnectedIdentities: React.FC<UserDetailsConnectedIdentitiesPro
     };
   }, [locale, identities]);
 
+  const onRemoveClicked = useCallback(
+    (identityID: string, identityName: string) => {
+      setConfirmationDialogData({
+        identityID,
+        identityName,
+      });
+    },
+    [setConfirmationDialogData]
+  );
+
+  const onDismissConfirmationDialog = useCallback(() => {
+    setConfirmationDialogData(null);
```

Comment:
During the dismiss animation of confirmation dialog, the identity name (email) disappears and cause strange layout. We should use a separate flag to control visibility of dialog. For example:
- `isHidden`: initially true, set to false when presenting, set to true when dismissing.
- `dialogData` initially an empty value, set to correct value when presenting, do not change when dismissing.